core/tests/performance_tests/base.py

```diff
@@ -28,6 +30,21 @@ class TestBase(unittest.TestCase):
 
     BASE_URL = 'http://localhost:9501'
 
+    LIBRARY_URL_SUFFIX = '/library'
+    EDITOR_URL_SLICE = '/create/'
+    PLAYER_URL_SLICE = '/explore/'
+    PROFILE_URL_SLICE = '/profile/'
+    LOGIN_URL_SUFFIX = '/_ah/login'
+    ADMIN_URL_SUFFIX = '/admin'
+    CONTINUE_URL_SUFFIX = (
+        '?continue=http%3A//localhost%3A9501/signup%3Freturn_url%3D%252F')
```

Comment:
Shouldn't the continue URL have BASE_URL in it? Seems like that localhost is being duplicated here, which means if we change BASE_URL in the future this won't work.